### PR TITLE
Avidemux 2.7.8

### PR DESCRIPTION
**Pull request by Dennis14e**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25054)

### CODE REVIEW

File: manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Avidemux.Avidemux
+PackageVersion: 2.7.8
```
Comment (ImJoakim) [inline manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.installer.yaml:5]: image

```suggestion
PackageVersion: 2.7.8.210306
```

File: manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Avidemux.Avidemux
+PackageVersion: 2.7.8
```
Comment (ImJoakim) [inline manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 2.7.8.210306
```

File: manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Avidemux.Avidemux
+PackageVersion: 2.7.8
```
Comment (ImJoakim) [inline manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.yaml:5]: image

```suggestion
PackageVersion: 2.7.8.210306
```

File: manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  Architecture: x64
+  InstallerType: nullsoft
+  InstallModes:
+    - interactive
+    - silent
+    - silentWithProgress
+  InstallerSuccessCodes:
+    - 1223
+  InstallerUrl: https://sourceforge.net/projects/avidemux/files/avidemux/2.7.8/avidemux_2.7.8__win64.exe/download
+  InstallerSha256: 5B6197FC98A356E1D3A15163C0813F3AEE7D35FB43D2D8FBE29F5E3A1596A818
```
Comment (ImJoakim) [inline manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.installer.yaml:18]: Let's add the ProductCode as there is no Publisher.

```suggestion
  InstallerSha256: 5B6197FC98A356E1D3A15163C0813F3AEE7D35FB43D2D8FBE29F5E3A1596A818
  ProductCode: "Avidemux 2.7 - 64 bits (64-bit)"
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
Please see my comments above. You also need to change the version folder:

`manifests/a/Avidemux/Avidemux/2.7.8/` -> `manifests/a/Avidemux/Avidemux/2.7.8.210306/`

https://github.com/microsoft/winget-pkgs/pull/25054#discussion_r688994278
https://github.com/microsoft/winget-pkgs/pull/25054#discussion_r688994291
https://github.com/microsoft/winget-pkgs/pull/25054#discussion_r688994292
https://github.com/microsoft/winget-pkgs/pull/25054#discussion_r688994294

**Comment by Dennis14e** [comment]
@ImJoakim Please review again.

**Comment by ImJoakim** [comment]
@palenshus @jamespik will take a look, they will most likely add a waiver. Example: #24527. Waiver was added for this PR #24482.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**